example/android/app/src/main/AndroidManifest.xml

```diff
@@ -7,6 +7,7 @@
     -->
     <uses-permission android:name="android.permission.INTERNET"/>
     <uses-permission android:name="android.permission.CAMERA" />
+    <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
```

Comment:
Given this is an example, can you add a comment to this line to let developer know this is only required to support API level 28 or above?

Reply:
Done